plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -1,5 +1,7 @@
+from __future__ import absolute_import
```

Comment:
I got curious why this is needed, is Pulp supporting Python older than 2.6? I am asking because I've found this `Changed in version 2.6: Always uses absolute imports.` on the documentation [1]

https://docs.python.org/2/c-api/import.html